src/integrations/prefect-kubernetes/integration_tests/src/prefect_kubernetes_integration_tests/test_eviction.py

```diff
@@ -20,7 +22,10 @@ async def test_pod_eviction(work_pool_name: str, capsys: pytest.CaptureFixture[s
         entrypoint=flow_entrypoint,
         name=flow_name,
         work_pool_name=work_pool_name,
-        job_variables={"image": "prefecthq/prefect:3-latest"},
+        job_variables={
+            "image": "prefecthq/prefect:3.2.11-python3.12",
+            "env": {"PREFECT_API_URL": "http://172.17.0.1:4200/api"},
```

Comment:
This is the Docker gateway IP address, which allows communication with the host from inside containers. I saw flow run failures because the pod couldn't reach the server.

Reply:
aha, I was gonna ask 🫡